packaging/linux/tuxbot/init

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
this is the script that prompts user to log into the keybase user and decrypt code key, not done yet